default_app_config = 'providers.org.biorxiv.apps.AppConfig'

"""
Example Record

<head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">

    <meta name="type" content="article">
    <meta name="category" content="research-article">

    <meta name="HW.identifier" content="/biorxiv/early/2016/01/27/038018.atom">
    <meta name="HW.pisa" content="biorxiv;038018v1">

    <meta name="DC.Format" content="text/html">
    <meta name="DC.Language" content="en">
    <meta name="DC.Title" content="Seqping: Gene Prediction Pipeline for Plant Genomes using Self-Trained Gene Models and Transcriptomic Data">
    <meta name="DC.Identifier" content="10.1101/038018">
    <meta name="DC.Date" content="2016-01-27">
    <meta name="DC.Publisher" content="Cold Spring Harbor Labs Journals">
    <meta name="DC.Rights" content="Â© 2016, Published by Cold Spring Harbor Laboratory Press. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/">
    <meta name="DC.AccessRights" content="restricted">
    <meta name="DC.Description" content="Although various software are available for gene prediction, none of the currently available gene-finders have a universal Hidden Markov Models (HMM) that can perform gene prediction for all organisms equally well in an automatic fashion. Here, we report an automated pipeline that performs gene prediction using self-trained HMM models and transcriptomic data. The program processes the genome and transcriptome sequences of a target species through GlimmerHMM, SNAP, and AUGUSTUS training pipeline that ends with the program MAKER2 combining the predictions from the three models in association with the transcriptomic evidence. The pipeline generates species-specific HMMs and is able to predict genes that are not biased to other model organisms. Our evaluation of the program revealed that it performed better than the use of the closest related HMM from a standalone program.">
    <meta name="DC.Contributor" content="Kuang-Lim Chan">
    <meta name="DC.Contributor" content="Rozana Rosli">
    <meta name="DC.Contributor" content="Tatiana Tatarinova">
    <meta name="DC.Contributor" content="Michael Hogan">
    <meta name="DC.Contributor" content="Mohd Firdaus-Raih">
    <meta name="DC.Contributor" content="Eng-Ti Leslie Low">

    <meta name="og:title" content="Seqping: Gene Prediction Pipeline for Plant Genomes using Self-Trained Gene Models and Transcriptomic Data">
    <meta name="og:url" content="http://biorxiv.org/content/early/2016/01/27/038018">
    <meta name="og:site_name" content="bioRxiv">
    <meta name="og:description" content="Although various software are available for gene prediction, none of the currently available gene-finders have a universal Hidden Markov Models (HMM) that can perform gene prediction for all organisms equally well in an automatic fashion. Here, we report an automated pipeline that performs gene prediction using self-trained HMM models and transcriptomic data. The program processes the genome and transcriptome sequences of a target species through GlimmerHMM, SNAP, and AUGUSTUS training pipeline that ends with the program MAKER2 combining the predictions from the three models in association with the transcriptomic evidence. The pipeline generates species-specific HMMs and is able to predict genes that are not biased to other model organisms. Our evaluation of the program revealed that it performed better than the use of the closest related HMM from a standalone program.">
    <meta name="og:type" content="article">

    <meta name="article:published_time" content="2016-01-27">
    <meta name="article:section" content="New Results">

    <meta name="citation_journal_title" content="bioRxiv">
    <meta name="citation_publisher" content="Cold Spring Harbor Labs Journals">
    <meta name="citation_title" content="Seqping: Gene Prediction Pipeline for Plant Genomes using Self-Trained Gene Models and Transcriptomic Data">
    <meta name="citation_publication_date" content="2016/01/01">
    <meta name="citation_mjid" content="biorxiv;038018v1">
    <meta name="citation_id" content="038018v1">
    <meta name="citation_public_url" content="http://biorxiv.org/content/early/2016/01/27/038018">
    <meta name="citation_abstract_html_url" content="http://biorxiv.org/content/early/2016/01/27/038018.abstract">
    <meta name="citation_full_html_url" content="http://biorxiv.org/content/early/2016/01/27/038018.full">
    <meta name="citation_pdf_url" content="http://biorxiv.org/content/biorxiv/early/2016/01/27/038018.full.pdf">
    <meta name="citation_doi" content="10.1101/038018">
    <meta name="citation_section" content="New Results">
    <meta name="citation_firstpage" content="038018">

    <meta name="citation_author" content="Kuang-Lim Chan">
    <meta name="citation_author_institution" content="Malaysian Palm Oil Board;">
    <meta name="citation_author_email" content="chankl@mpob.gov.my">
    <meta name="citation_author" content="Rozana Rosli">
    <meta name="citation_author_institution" content="Malaysian Palm Oil Board;">
    <meta name="citation_author" content="Tatiana Tatarinova">
    <meta name="citation_author_institution" content="University of Southern California;">
    <meta name="citation_author" content="Michael Hogan">
    <meta name="citation_author_institution" content="Orion Genomics;">
    <meta name="citation_author" content="Mohd Firdaus-Raih">
    <meta name="citation_author_institution" content="Universiti Kebangsaan Malaysia">
    <meta name="citation_author" content="Eng-Ti Leslie Low">
    <meta name="citation_author_institution" content="Malaysian Palm Oil Board;">

    <meta name="citation_date" content="2016-01-27">

    <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution">
    <meta name="generator" content="Drupal 7 (http://drupal.org)">

</head>
"""
